src/agent.rs

```diff
@@ -695,4 +707,20 @@ mod tests {
         let _agent: Box<dyn Send> = Box::new(AgentBuilder::new().build());
         let _agent: Box<dyn Sync> = Box::new(AgentBuilder::new().build());
     }
+
+    #[test]
+    fn agent_config_debug() {
+        let agent = AgentBuilder::new().build();
+        assert_eq!(
+            format!("{:?}", agent),
+            "Agent { config: AgentConfig { \
+            proxy: None, timeout_connect: Some(30s), timeout_read: None, \
+            timeout_write: None, timeout: None, https_only: false, \
+            no_delay: true, redirects: 5, redirect_auth_headers: Never, \
+            user_agent: \"ureq/2.4.0\", tls_config: TlsConfig \
```

Comment:
Will it not be tedious to lock down the version in a test like this?

Reply:
Oops, good point. I will modify the test to not be so over-specified.